actors/miner/tests/util.rs

```diff
@@ -1108,6 +1085,187 @@ impl ActorHarness {
         }
     }
 
+    pub fn prove_commit_sectors2(
```

Comment:
It was convenient to add this here rather than a new file or prove_commit.rs to make use of the same helper functions without figuring out a clear way to share functions.  It might makes sense to group the new tests in this one file and rename.

Reply:
Does this comment belong with your changes in prove_replica_test.rs? I was going to ask you to extract to a new file. Meanwhile this comment and the helpers _are_ in the giant util.rs, and for now I think that's fine. (I extracted a few more bits here in #1407).